index.ts

```diff
@@ -113,40 +140,39 @@ function candidateToGeojson (candidate) {
  * @param {string} [$0.url]                       optional URL to override ESRI suggest endpoint
  * @return {Promise}                              A Promise that'll get resolved with the suggest result
  */
-export function autocomplete ({
+export function autocomplete({
+  boundary,
   clientId,
   clientSecret,
-  boundary,
   focusPoint,
   text,
   url
 }: BaseQuery & {
-  boundary?: Boundary,
-  focusPoint?: any,
-  text: string,
+  boundary?: Boundary
+  focusPoint?: Point
+  text?: string
 }): Promise<BaseResponse> {
```

Comment:
The `BaseResponse` is not even correct here as the query part is an Object with the text key/value. Please update the return type.

Reply:
Fixed in cdd90839c90a85888732fb6718964b8ddd8fb1b6